datadog/api/resources.py

```diff
@@ -99,7 +99,7 @@ class UpdatableAPIResource(object):
     Updatable API Resource
     """
     @classmethod
-    def update(cls, id, params=None, **body):
+    def update(cls, id, synthetics=False, params=None, **body):
```

Comment:
Is this the best design or wouldn't it be better to have an `UpdatableAPISyntheticsResource` class that is being used instead?
I'm afraid that special casing here with keyword arguments never ends up with some new API coming up.